app/retail/templates/emails/bounty/no_applicant_reminder.html

```diff
@@ -77,11 +77,11 @@
 </style>
 
 <div class="bounty-box">
-  <h1>Get more applicants on your bounty</h2>
+  <h1>Get more applicants on your bounty!</h2>
 
   <p class="mb-0 mt-4">We noticed you dont have any contributors on your bounty.</p>
   <p class="mt-1">
-    Go to the Users Directory to discover and invite the best contributors
+    Go to the User Directory to discover and invite the best contributors
```

Comment:
would we wanna wrap this within a link @frankchen07  ?

```html
<p class="mt-1">
    Go to the 
    <a  href="https://gitcoin.co/users/" target="_blank">User Directory </a>
    to discover and invite the best contributors for your bounty!
</p>```

Reply:
I think is not necessary since the button with the link is behind this

Reply:
agree, there's a button with the directory link in later lines (113-114)